src/fiat/bank-authorities.md

```diff
@@ -30,51 +36,43 @@ curl -X POST "https://service.centrapay.com/api/bank-authorities" \
   -d '{
     "fullName": "John Doe",
     "accountId": "Jaim1Cu1Q55uooxSens6yk",
-    "streetAddress": "11 Shortland St",
-    "suburb": "Auckland Central",
-    "city": "Auckland",
-    "postCode": "1100",
+    "userAuthorizesDirectDebit": true,
     "phoneNumber": "+64212345678",
     "emailAddress": "John.doe@email.com",
     "bankAccountNumber": "12-1234-1234567-123",
     "bankAccountName": "John Doe"
   }'
 ```
 
+<span id="bank-fields-anchor"></span>
 **Required Fields**
 
 |       Field       |  Type  |                    Description                    |
 | :---------------- | :----- | :------------------------------------------------ |
-| fullName          | String | The first and last name of the user               |
+| accountId         | String | The account Id                                    |
 | phoneNumber       | String | The user's phone number                           |
-| emailAddress      | String | The user's email address                          |
 | bankAccountNumber | String | The user's bank account number                    |
 | bankAccountName   | String | The name on the bank account provided by the user |
 
 **Optional Fields**
 
-|       Field       |  Type  |                    Description                    |
-| :---------------- | :----- | :------------------------------------------------ |
-| streetAddress     | String | The street address of the user                    |
-| suburb            | String | The suburb relating to the user's address         |
-| city              | String | The city relating to the users address            |
-| postCode          | String | The postal code relating to the user's address    |
+Note, fields which have a star (✩) are required for Top Up.
+
+|       Field               |  Type   |                    Description                                                       |
+| :------------------------ | :------ | :----------------------------------------------------------------------------------- |
+| userAuthorizesDirectDebit | Boolean | ✩ The user accepts our Direct Debit terms and has authority to operate this account. |
```

Comment:
Should we call this "directDebitAuthorized"? What will the default value be? False makes more sense but is a breaking change.

Reply:
Yeah, default should be false.

I was keen on making the subject of this field the user because I wanted to make it pretty clear that calls with this property set were something the account owner had seen and approved of. While we could always point at our docs if there was confusion, making it super clear from the name makes me feel better about scripting.

Apart from making the field shorter, are there other reasons we'd change this name?

It feels like we should link to our DD terms form here too.

Reply:
I don't like it. At the moment user authority is implied. I don't think there needs to be extra clarity.

Reply:
I've updated this to `directDebitAuthorized`